internal/kibana/outputs.go

```diff
@@ -0,0 +1,64 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package kibana
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// Output represents an Output in Fleet.
+type Output struct {
+	ID                  string   `json:"id,omitempty"`
+	Name                string   `json:"name"`
+	Type                string   `json:"type"`
+	IsDefault           bool     `json:"is_default"`
+	IsDefaultMonitoring bool     `json:"is_default_monitoring"`
+	Hosts               []string `json:"hosts"`
+}
+
+// CreateOutput persists the given Output in Fleet.
+func (c *Client) CreateOutput(o Output) (*Output, error) {
+	reqBody, err := json.Marshal(o)
+	if err != nil {
+		return nil, errors.Wrap(err, "could not convert output (request) to JSON")
+	}
+
+	statusCode, respBody, err := c.post(fmt.Sprintf("%s/outputs", FleetAPI), reqBody)
```

Comment:
Are these outputs removed after the benchmark ?

Reply:
I ended up not needing the outputs, will delete the file